<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWidget</class>
 <widget class="QWidget" name="SetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="measurementTypeLabel">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="measurementType">
        <item>
         <property name="text">
          <string>Open Circuit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electrolysis</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear Sweep</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyclic Voltammetry</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="openCircuitWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="openCircuitDuration">
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="electrolysisWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_7" columnstretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Current range:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="electrolysisPotential">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="minimum">
         <number>-2000</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="electrolysisDuration">
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Potential:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="electrolysisCurrentRange">
          <item>
           <property name="text">
            <string>±200 nA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±1 µA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±100 µA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±10 mA</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="electrolysisAutoRange">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="linearSweepWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_8" columnstretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Scan rate:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="linearSweepEndPotential">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="minimum">
         <number>-2000</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="linearSweepScanRate">
        <property name="suffix">
         <string> mV/s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current range:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>End potential:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Start potential:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="linearSweepStartPotential">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="minimum">
         <number>-2000</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>-1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="linearSweepCurrentRange">
          <item>
           <property name="text">
            <string>±200 nA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±1 µA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±100 µA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±10 mA</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="linearSweepAutoRange">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="cyclicVoltammetryWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_9" columnstretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <widget class="QSpinBox" name="cyclicVoltammetryVertex1">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="minimum">
         <number>-2000</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>-1000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="cyclicVoltammetryVertex2">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="minimum">
         <number>-2000</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Scan rate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Current range:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="cyclicVoltammetryCycles">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="cyclicVoltammetryScanRate">
        <property name="suffix">
         <string> mV/s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Vertex 1:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Cycles:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Vertex 2:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Start potential:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="cyclicVoltammetryStartPotential">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="minimum">
         <number>-2000</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="cyclicVoltammetryCurrentRange">
          <item>
           <property name="text">
            <string>±200 nA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±1 µA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±100 µA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±10 mA</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cyclicVoltammetryAutoRange">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>measurementType</tabstop>
  <tabstop>openCircuitDuration</tabstop>
  <tabstop>electrolysisPotential</tabstop>
  <tabstop>electrolysisDuration</tabstop>
  <tabstop>linearSweepCurrentRange</tabstop>
  <tabstop>linearSweepAutoRange</tabstop>
  <tabstop>linearSweepScanRate</tabstop>
  <tabstop>linearSweepStartPotential</tabstop>
  <tabstop>linearSweepEndPotential</tabstop>
  <tabstop>cyclicVoltammetryCurrentRange</tabstop>
  <tabstop>cyclicVoltammetryAutoRange</tabstop>
  <tabstop>cyclicVoltammetryScanRate</tabstop>
  <tabstop>cyclicVoltammetryStartPotential</tabstop>
  <tabstop>cyclicVoltammetryVertex1</tabstop>
  <tabstop>cyclicVoltammetryVertex2</tabstop>
  <tabstop>cyclicVoltammetryCycles</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>linearSweepAutoRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>linearSweepCurrentRange</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cyclicVoltammetryAutoRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>cyclicVoltammetryCurrentRange</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
